pkg/oidc/token_request.go

```diff
@@ -74,8 +74,8 @@ type AccessTokenRequest struct {
 	ClientID            string `schema:"client_id"`
 	ClientSecret        string `schema:"client_secret"`
 	CodeVerifier        string `schema:"code_verifier"`
-	ClientAssertion     string `schema:"client_assertion"`
-	ClientAssertionType string `schema:"client_assertion_type"`
+	ClientAssertion     string `schema:"client_assertion,omitempty"`
+	ClientAssertionType string `schema:"client_assertion_type,omitempty"`
```

Comment:
For consistency we should add `omitempty` to all optional fields, like the `client_secret` and `code_verifier`. There might be a chance OP like okta would fail as well, if one would send an assertion and the secret will be an empty value in that case.

Reply:
@muir I'm not seeing it. Did you forget to commit or push?

Reply:
Hmm, I'm not sure what happened. Pushing, again.